Ghidra/Processors/PowerPC/data/languages/ppc_vle.sinc

```diff
@@ -953,3 +953,76 @@ IMM16B: val						is IMM_0_10_VLE & IMM_16_20_VLE [ val = (IMM_16_20_VLE << 11) |
 :se_isync						is $(ISVLE) & OP16_VLE=1 {
 	instructionSynchronize();
 }
+
+# =================================================================
+# Saturating Arithmetic Instructions
+
+# e_addwus - Add Word Unsigned Saturate
+:e_addwus D,A,B					is $(ISVLE) & OP=31 & D & A & B & XOP_1_10=0x36A & BIT_0=0
+{
+	tmpA:4 = A:4;
+	tmpB:4 = B:4;
+	tmpResult:4 = tmpA + tmpB;
+	if (!carry(tmpA, tmpB)) goto <no_overflow>;
```

Comment:
Just like the other pull-request, this should be handled without the use of gotos.

```
local cond:1 = carry(tmpA, tmpB);
D = (zext(cond) * 0xffffffff) + (zext(!cond) * tmpResult);
```

and similarly for the other instructions.